packages/ia-components/components/audio-players/audio-player.less

```diff
@@ -16,6 +16,7 @@
   }
 
   .content-window {
+    height: 440px;
```

Comment:
SUGGESTION/QUESTION: Significance of this magic number? I see it repeated below. Should it be a variable?